conf/airframes/tudelft/bebop_autonomous_race_2018.xml

```diff
@@ -257,7 +259,7 @@
   <section name="AUTOPILOT">
     <define name="MODE_STARTUP" value="AP_MODE_NAV"/>
     <define name="MODE_MANUAL" value="AP_MODE_ATTITUDE_DIRECT"/>
-    <define name="MODE_AUTO1" value="AP_MODE_ATTITUDE_Z_HOLD"/>
```

Comment:
The mode `AP_MODE_ATTITUDE_Z_HOLD` still exists with the static autopilot, but not in the generated one that you have selected for this airframe. It can of course be added or copied from https://github.com/paparazzi/paparazzi/blob/master/conf/autopilot/rotorcraft_autopilot.xml#L68

Reply:
So the problem is that you can not at the same time have ```AP_MODE_MODULE``` and ```AP_MODE_ATTITUDE_Z_HOLD``` correct?